FaunaDB.Client/Types/Codec.cs

```diff
@@ -47,6 +47,22 @@ public static IResult<Value> VALUE(Value value)
 
         public static IResult<IReadOnlyDictionary<string, Value>> OBJECT(Value input) =>
             Cast.MapTo<ObjectV, IReadOnlyDictionary<string, Value>>(input, x => x.Value);
+
+        public static IResult<T> DECODE<T>(Value input)
```

Comment:
No conventions was used here, it's something I did wrong in the beginning, now we have to live with it or create siblings methods with correct names and deprecate the old ones